Snowflake.Data.Tests/UnitTests/SFArrowResultChunkTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * Copyright (c) 2012-2019 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.Linq;
+using Apache.Arrow;
+using NUnit.Framework;
+using Snowflake.Data.Core;
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    [TestFixture]
+    class SFArrowResultChunkTest
+    {
+        [Test]
+        [Ignore("SFArrowResultChunkTest")]
+        public void SFArrowResultChunkTestDone()
+        {
+            // Do nothing;
```

Comment:
could we change this comment to say for what we need it? e.g. test progress marker